dotcom-rendering/src/lib/decidePalette.ts

```diff
@@ -550,8 +550,7 @@ const textYoutubeOverlayKicker = (format: ArticleFormat) => {
 	}
 };
 
-const backgroundDynamoSublink = (_format: ArticleFormat): string =>
-	palette.neutral[97];
+const backgroundDynamoSublink = (): string => palette.neutral[97];
```

Comment:
Same as above, we should turn this into a const. We can take this as a follow-up.